docs-src/cloud/tcld/generate-certificates/end-entity-certificate.md

```diff
@@ -0,0 +1,100 @@
+---
+id: end-entity-certificate
+title: tcld generate-certificates end-entity-certificate
+sidebar_label: end-entity-certificate
+description: Generate end-entity (leaf) certificates with tcld.
+tags:
+  - tcld
+  - cli-reference
+---
+
+The `tcld generate-certificates end-entity-certificate` command generates end-entity (leaf) certificates for Temporal Cloud.
+
+`tcld generate-certificates end-entity-certificate <modifiers>`
+
+Alias: `leaf`
+
+The following modifiers control the behavior of the command.
+
+#### --organization
+
+Specify an organization name for certificate generation.
+
+Alias: `--org`
+
+**Example**
+
+```bash
+tcld generate-certificates end-entity-certificate --organization <value>
+```
+
+#### --organizaation-unit
+
+Specify the name of the organization unit (optional).
```

Comment:
```suggestion
Optional: Specify the name of the organization unit.
```
Per https://developers.google.com/style/procedures#optional-steps, although IIRC we specify required modifiers rather than optional modifiers.